game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package games.strategy.engine.framework.map.download;
+// TODO: move to package games.strategy.engine.framework.map.download.client
+
+import java.util.function.Supplier;
+
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+/**
+ * Provides methods to download files via HTTP.
+ */
+public final class DownloadConfiguration {
+
+  private static ContentReader contentReader;
+  private static DownloadLengthReader downloadLengthReader;
```

Comment:
It looks like these fields can be `final`.

Reply:
5dd6e76ab